package.json

```diff
@@ -58,7 +58,6 @@
     "eslint-plugin-promise": "^4.2.1",
     "eslint-plugin-standard": "^4.0.1",
     "glsl-transpiler": "^1.8.5",
-    "jasmine": "^2.4.1",
     "jest": "^24.9.0",
     "jest-image-snapshot": "^2.12.0",
     "jest-puppeteer": "^4.4.0",
```

Comment:
Are there any other dependencies we could remove e.g. jsdom? Happy to see this raised as a separate issue

Reply:
Not that I'm aware of. `jsdom` is still used in the various `bundleSpec` tests that most packages have, e.g. in d3fc-annotation:

https://github.com/d3fc/d3fc/blob/bb3958f2df147e4614a4d8289a9819d8e8e7145c/packages/d3fc-annotation/test/bundleSpec.js#L1

I've opened https://github.com/d3fc/d3fc/issues/1591 which lists a few potential areas for improvement for the bundle specs, including potentially using Jest's `jsdom` [`testEnvironment`](https://jestjs.io/docs/en/configuration#testenvironment-string) and removing the explicit `jsdom` dependency.